src/components/CatalogCard/CatalogCard.tsx

```diff
@@ -140,24 +142,14 @@ export const CatalogCard: FunctionComponent<CatalogCardProps> = ({
             {publishDate}
           </Text>
 
-          {typeof author === "string" ? (
-            <Text data-testid={testIds.author} fontSize="sm" isTruncated>
-              {author}
-            </Text>
-          ) : author.url ? (
-            <ExternalLink
-              data-testid={testIds.author}
-              fontSize="sm"
-              href={author.url}
-              onClick={(e) => e.stopPropagation()}
-            >
-              {author.name}
-            </ExternalLink>
-          ) : (
-            <Text data-testid={testIds.author} fontSize="sm" isTruncated>
-              {typeof author === "string" ? author : author.name}
-            </Text>
-          )}
+          <UILnk
+            color="blue.500"
+            fontSize="sm"
+            href={`./search?q=${authorName}`}
+            onClick={(e) => e.stopPropagation()}
+          >
```

Comment:
```suggestion
          <UILink
            as={Link}
            color="blue.500"
            fontSize="sm"
            to={getSearchPath({ query: authorName })}
          >
```

Reply:
getSearchPath will need to be imported from the same file as `getPackagePath` (which is already imported)

Reply:
Done